### DISCUSSION TITLE
Display just the individual attributes

### DISCUSSION THREAD

**Discussion by weronikazak** [dicebear/dicebear#353] [Q&A] [answered]
Hey, I'm using Micah avatars in my React Native app. Is there a way for me to display only the individual attributes without the entire model? For example, just the eyes without the rest of the face. Thanks in advance!

**Comment by FlorianKoerner** [comment]
HI @weronikazak 

Unfortunately, this is not possible for the avatar style "micah" at the moment. But I'll keep that in mind for the next major version.

**Comment by FlorianKoerner** [comment, marked as answer]
I looked at it again and realized that my answer was wrong. It is already possible. Here are a few examples:

![](https://api.dicebear.com/7.x/micah/svg?base[]&hair[]&shirt[]&ears[]&glassesProbability=0&backgroundColor=F9C9B6,AC6651,77311D&scale=200&translateX=-5&translateY=8&seed=example&size=128)

HTTP-API
```
https://api.dicebear.com/7.x/micah/svg?base[]&hair[]&shirt[]&ears[]&glassesProbability=0&backgroundColor=F9C9B6,AC6651,77311D&scale=200&translateX=-5&translateY=8&seed=example
```

JS-Library
```js
import { createAvatar } from '@dicebear/core';
import * as micah from '@dicebear/micah';

const avatar = createAvatar(micah, {
  base: [],
  hair: [],
  ears: [],
  glassesProbability: 0,
  backgroundColor: ["F9C9B6", "AC6651", "77311D"],
  scale: 200,
  translateX: -5,
  translateY: 8,
  seed: 'example'
});
```
